src/app/zap-templates/attribute-size.zapt

```diff
@@ -6,6 +6,6 @@
 // ZCL attribute sizes
 {{#zcl_atomics}}
 {{#if size}}
-{{ident}}ZCL_{{asDelimitedMacro name}}_ATTRIBUTE_TYPE, {{size}}, \
```

Comment:
I do not see any modification in the `/src/app/` directory but if none were made the `\` shouldn't be removed since the attribute size were used as element of a macro array

Reply:
It needs to be removed otherwise the code won't compile. It throws a warning and warnings ends up as errors within CHIP.